app/scripts/notice-controller.js

```diff
@@ -27,36 +27,36 @@ module.exports = class NoticeController extends EventEmitter {
     }
   }
 
-  setNoticesList(list) {
+  setNoticesList (list) {
     var data = this.configManager.getData()
     data.noticesList = list
     this.configManager.setData(data)
     return Promise.resolve(true)
   }
 
-  markNoticeRead(notice, cb) {
-    cb = cb || function(err){ if (err) throw err }
+  markNoticeRead (notice, cb) {
+    cb = cb || function (err) { if (err) throw err }
```

Comment:
Furthermore, all this linting is interfering with my ability to easily see what you actually changed in this PR.

Maybe we should just update the lintfile and all outlying instances of this stuff at once, in its own PR.